stages/org.osbuild.kickstart

```diff
@@ -170,6 +170,25 @@ def make_network(options: Dict) -> List[str]:
     return res
 
 
+def make_post(post_list):
+    res = []
+    for post in post_list:
+        start = "%post"
+        if post.get("erroronfail"):
+            start += " --erroronfail"
+        if post.get("nochroot"):
+            start += " --nochroot"
+        log = post.get("log")
+        if log:
+            start += f" --log \"{log}\""
```

Comment:
Nitpick: You could have used simple quotes to avoid having to escape the double quote. I find non-escaped strings much easier to read.